### ISSUE TITLE
Runway Management Improvement

### ISSUE THREAD

**Issue by scott-j-smith** [openscope/openscope#300]
Hi all,

I am running in 4.1.2 and I noticed by accident that you can ignore the potential for collisions on the runway between arriving and departing flights.  I started a takeoff just as a plane was landing and didn't notice it because the red arrival box and dot were behind the blue ones of my selected (and thus highlighted) flight.  But no warnings or collisions happened.

The game will detect issues between planes that are either both arriving or both departing if they are too close on the runway, but not if they are opposite.   So you can have a plane landing and taking off at the same time, even if the dots have converged, both show altitude of 000, etc.  Obviously this can't happen in real life, so adding an improvement to the game to catch this would be more realistic.  This can happen at any airport as far as I can tell.

Thanks,
Scott

**Comment by n8rzz** [comment]
Yep, this appears to be a bug.  We may have addressed this, by happenstance, in #139 ( #193 ).  so we should check that on openscope-dev-pr-193.herokuapp.com to be sure.

If not, this is a legit defect that should be addressed.

**Comment by n8rzz** [comment]
could possibly relate to future work on #93

**Comment by erikquinn** [comment]
Definitely wouldn't be fixed by #139. Actually, somewhere in the conflict detection logic, we consciously disabled the possibility of any conflicts/violations/collisions with aircraft on the ground (because the conflict detection logic is fairly simple, and does not have rule sets for judging separation of aircraft on runways, taxiways, etc.

**Comment by n8rzz** [comment]
this should actually be done under #314
